transports/msmq/distributor/error-handling.md

```diff
@@ -30,7 +30,7 @@ Immediate Retries are always performed on the worker.
 
 The behavior of Delayed Retries is different between major versions.
 
-Delayed Retries has two configurable items TimeIncrease and NumberOfRetries.
+Delayed Retries has two configurable items *TimeIncrease* and *NumberOfRetries*.
```

Comment:
i think it means something like  "any change in distributor configuration needs to be replicated to all nodes"?